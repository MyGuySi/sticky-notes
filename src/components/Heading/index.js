// @flow

import * as React from 'react';
import styles from './styles.module.css';

type Props = {
  level?: 1 | 2 | 3 | 4 | 5 | 6,
  children: React.Node
}

function Heading (props: Props) {
  let Element = 'h1';
  if (props.level) {
    Element = 'h' + props.level;
  }
  return (
    <Element className={styles.heading}>{props.children}</Element>
  )
}

export default Heading