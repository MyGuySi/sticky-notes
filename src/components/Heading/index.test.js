// @flow

import React from 'react';
import renderer from 'react-test-renderer'
import Heading from './index'

test('Renders correctly', () => {
  const component = renderer.create(
    <Heading>Test</Heading>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})