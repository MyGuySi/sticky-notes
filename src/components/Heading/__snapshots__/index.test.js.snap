// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<h1
  className="heading"
>
  Test
</h1>
`;
